main.go

```diff
@@ -114,13 +114,18 @@ func ReportSubcommand() {
 }
 
 func main() {
-	// TODO(#6): index out of range error when no subcommands are provided
-	switch os.Args[1] {
-	case "list":
-		ListSubcommand()
-	case "report":
-		ReportSubcommand()
-	default:
-		panic(fmt.Sprintf("`%s` unknown command", os.Args[1]))
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "list":
+			ListSubcommand()
+		case "report":
+			ReportSubcommand()
+		default:
+			panic(fmt.Sprintf("`%s` unknown command", os.Args[1]))
+		}
+	} else {
+		//TODO (#7) implement a map for options instead of println'ing them all there
```

Comment:
Please, check out the format of [reported TODOs](https://github.com/tsoding/snitch#reported-todo) It should look like

```
// TODO(#7): implement a map for options instead of println'ing them all there
```

Also, you please leave [unreported TODOs](https://github.com/tsoding/snitch#unreported-todo) I'll report all of them myself once the PR is approved. This is because some TODOs may be eliminated or changed during the review. This one is fine, though. Let's keep it. :+1: